modules/configuring-firewall.adoc

```diff
@@ -189,15 +189,15 @@ Alternatively, if you choose to not use a wildcard for AWS APIs, you must includ
 |443
 |Used to install and manage clusters in an AWS environment.
 
-|`s3.amazonaws.com`
+|`*.s3.amazonaws.com`
 |443
 |Used to install and manage clusters in an AWS environment.
 
-|`s3.<aws_region>.amazonaws.com`
+|`*s3.<aws_region>.amazonaws.com`
 |443
 |Used to install and manage clusters in an AWS environment.
 
-|`s3.dualstack.<aws_region>.amazonaws.com`
+|`*s3.dualstack.<aws_region>.amazonaws.com`
```

Comment:
```suggestion
|`*.s3.dualstack.<aws_region>.amazonaws.com`
```
I believe that with-the-period is correct. Best-practice to have QE look before peer review, IMHO.

Reply:
yes, it supposed to be `*.s3.dualstack.<aws_region>.amazonaws.com`

Reply:
@gpei, Thanks for your comment. Can you review the updated wildcard syntax?